Added support for passing additional startup options and start timeout

borgeorgiev opened the pull request:
Extended plugin to support specifying additional mongodb startup options
on global and job level. 
Added a field for specifying custom start timeout on global and job
level.

Commit:
Extended plugin to support specifying additional mongodb startup options
on global and job level. 
Added a field for specifying custom start timeout on global and job
level.

buildhive commented:
[Jenkins » mongodb-plugin #6](https://buildhive.cloudbees.com/job/jenkinsci/job/mongodb-plugin/6/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

Commit:
Fix doCheckName - accidently the implementation was replaced with wrong
one

buildhive commented:
[Jenkins » mongodb-plugin #7](https://buildhive.cloudbees.com/job/jenkinsci/job/mongodb-plugin/7/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

Commit:
Fix java.io.NotSerializableException caused by using anonymous inner
class in FileCallable operation. This problem is prevening the user from
setting custom dbpath.

buildhive commented:
[Jenkins » mongodb-plugin #8](https://buildhive.cloudbees.com/job/jenkinsci/job/mongodb-plugin/8/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

kiy0taka commented:
Thanks.
